doc/en/user/source/community/features-templating/directives.rst

```diff
@@ -709,14 +709,20 @@ The :code:`flat_output` will act in the following way:
 
 JSON-LD
 """"""""
-A JSON-LD template can be defined as a GeoJSON template since it is a JSON based output as well. However it needs to have a :code:`@context` attribute, object or array at the beginning of it in orther to conform to the standard.
-To accomplish this requirement it is possible to specify the :code:`@context` as an :code:`option` in the template, like in the following one:
+A JSON-LD template can be defined as a GeoJSON template since it is a JSON based output as well. However it needs to have a :code:`@context` attribute, object or array at the beginning of it in orther to conform to the standard. Moreover each JSON Object must have an :code:`@type` defining a type through a vocabulary term.
+To accomplish this requirements it is possible to specify a bounch of :code:`$options` on the template:
+
+* :code:`@context` providing a full JSON-LD :code:`@context`.
+* :code:`@type` providing a type term for the root JSON object in the final output (by default the value is :code:`FeatureCollection`).
+* :code:`collection_name` providing an alternative name for the features array in the final output (by default :code:`features` is used). The option is usefull in case the user wants to use a features attribute name equals to a specific term defined in a vocabulary.
```

Comment:
```suggestion
* :code:`collection_name` providing an alternative name for the features array in the final output (by default :code:`features` is used). The option is useful in case the user wants to use a features attribute name equals to a specific term defined in a vocabulary.
```

Reply:
fixed